contracts/btc-staking/src/error.rs

```diff
@@ -50,6 +50,8 @@ pub enum ContractError {
     FinalityProviderNotFound(String),
     #[error("Staking tx hash already exists: {0}")]
     DelegationAlreadyExists(String),
+    #[error("BTC delegation is not active")]
+    DelegationIsNotActive,
```

Comment:
For clarity:
```suggestion
    #[error("BTC delegation is not active: {0}")]
    DelegationIsNotActive(String),
```
and use the staking tx hash or so for reference.